lua/completion.lua

```diff
@@ -101,6 +101,10 @@ local function hasConfirmedCompletion()
   if opt.get_option('enable_auto_paren') == 1 then
     autoAddParens(completed_item)
   end
+  if completed_item.user_data.lsp ~= nil
+	  and vim.g.completion_enable_snippet == "snippets.nvim" then
```

Comment:
I think keep the if statement together with the applyAdditionalTextEdits is slightly better. For ex
```
  if completed_item.user_data.lsp ~= nil then
    applyAddtionalTextEdits(completed_item)
    if vim.g.completion_enable_snippet = "snippet.nvim" then
       ...
    end
  end
```

Reply:
Makes sense, I'll change it!